modules/k8s/config_test.go

```diff
@@ -114,6 +114,7 @@ preferences: {}
 users:
 - name: minikube
   user:
+    as-user-extra: null
```

Comment:
For some reason, without this change, five k8s tests fail. I don't really understand why, but I've confirmed that they fail when run with go 1.18 but pass with go 1.17. Very strange.